.github/workflows/release.yaml

```diff
@@ -0,0 +1,157 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - release_workflow
+  workflow_dispatch:
+    inputs:
+      publish:
+        type: boolean
+        description: send packages to PyPi and the tlcpack wheels list
+
+jobs:
+  linux-pypi:
+    strategy:
+      fail-fast: false
+      matrix:
+        # matrix of build configs
+        config:
+          - cuda: 'none'
+            image: 'tlcpack/package-cpu:1d98bf9'
+            package_name: 'apache-tvm'
+          - cuda: '10.2'
+            image: 'tlcpack/package-cu102:1d98bf9'
+            package_name: 'apache-tvm'
+          - cuda: '11.3'
+            image: 'tlcpack/package-cu113:1d98bf9'
+            package_name: 'apache-tvm'
+          - cuda: '11.6'
+            image: 'tlcpack/package-cu116:1d98bf9'
+            package_name: 'apache-tvm'
```

Comment:
So for the CUDA packages I'm not sure we were planning to use `apache-tvm` as well. I think each of them need their own repo, similar to what `mxnet` does, e.g. `apache-tvm-cu102`, `apache-tvm-cu113`, `apache-tvm-cu116`

Apart from that, we can keep `apache-tvm` for the standard CPU package in all platforms.

https://pypi.org/project/mxnet/